api/utils/keys/yubikey.go

```diff
@@ -49,6 +49,8 @@ import (
 const (
 	// PIVCardTypeYubiKey is the PIV card type assigned to yubiKeys.
 	PIVCardTypeYubiKey = "yubikey"
+
+	pivAuthErrMessage = "smart card error 6982: security status not satisfied"
```

Comment:
This makes me a bit uneasy, matching strings is usually brittle. Is there something better we could do?

Reply:
The actual error code returned by the smart card is "6982", and the PIV library wraps that with the user readable message. Unfortunately the PIV library doesn't export any of the methods used. Would it be better to just match on "6982"?

Reply:
I considered that. Maybe split on `[\s:]` and see if the word is there? It's just an idea though.

Reply:
> split on [\s:]

What does this mean?

Reply:
Sorry, that was a bit cryptic. I meant split on spaces or `:`, as in: 

```go
re := regexp.MustCompile(`[\s:]`)
if slices.Contains(re.Split(err.Error(), "6982") {
  // ...
}
```

Reply:
Ah I see, I think just depending on the error code and no error structure will be the least brittle, we can require the error structure later if it's needed.